src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
```

Comment:
Your should use `this` to get value
```suggestion
    get info() {
      const {name, version, wheels} = this;
      
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },
```